sangsu/PSG_42587.py

```diff
@@ -0,0 +1,28 @@
+def solution(priorities, location):
+    answer = 0
+    new_priorities = []
+    for i in range(len(priorities)):
+        k = priorities.pop(0)
+        new_priorities.append((k, i))
```

Comment:
new_priorities = []
    for i in range(len(priorities)):
        new_priorities.append([priorities[i], i])